mass : 2.1
roll_time_constant  : 0.337975
roll_gain           : 0.881448
pitch_time_constant : 0.391407
pitch_gain          : 0.672181

drag_coefficients : [0.03, 0.03, 0]

q_position_x : 80
q_position_y : 80
q_position_z : 120

q_velocity_x : 30
q_velocity_y : 30
q_velocity_z : 40

q_attitude_pos : 20
q_attitude_vel : 20

r_command : [30, 30, 1]

roll_limit : 30
pitch_limit : 30
yaw_rate_limit : 5
thrust_min : 3
thrust_max : 30

k_yaw : 0.5

thrust_to_throttle : 0.505
throttle_limit : 0.85